package.json

```diff
@@ -14,19 +14,20 @@
   "author": "MasterBorn Sp. z o.o.",
   "license": "MIT",
   "dependencies": {},
-  "devDependencies": {
-    "eslint": "^7.7.0"
-  },
   "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.8.1",
+    "@typescript-eslint/parser": "^4.8.1",
     "@babel/core": "^7.11.1",
     "@babel/eslint-parser": "^7.11.3",
-    "eslint": "^7.7.0",
+    "eslint": "^7.14.0",
```

Comment:
**Q** Is it required? I would be opposed to bumping this up if that's not crucial due to the fact of current usage 🤔